pages/common/rip.md

```diff
@@ -0,0 +1,28 @@
+# rip
+
+> Remove files or directories by sending them to the graveyard, allowing for them to be recovered.
+> More information: <https://github.com/nivekuil/rip>.
+
+- Remove files or directories from specified locations and place them in the graveyard:
+
+`rip {{path/to/file_or_directory}} {{path/to/another/file_or_directory}}`
+
+- Interactively remove files or directories, with a prompt before every removal:
+
+`rip -i {{path/to/file_or_directory}} {{path/to/another/file_or_directory}}`
+
+- List all files and directories in the graveyard that were originally within the current directory:
+
+`rip -s`
+
+- Permanently delete every file and directory in the graveyard:
+
+`rip -d`
+
+- Put back the files and directories which were affected by the most recent removal:
+
+`rip -u`
+
+- Put back every file and directory that is listed by `rip -s`:
+
+`rip -su`
```

Comment:
```suggestion
`rip --seance --unbury`
```

Reply:
In my opinion, `--seance --unbury` is way to verbose compared to `-su`, and additionally, the name `--seance` is not the most helpful name for understanding what the option does. I think it would be clearer to leave it in the short form.

Of course, if you still disagree, I can make the change.

Reply:
I do agree, that the convention to use long options is always great, but it makes maintaining a lot easier. Also, most of the time it's easy to quickly find out the short version.